doc/releases/changelog-0.41.0.md

```diff
@@ -215,138 +221,170 @@ With `qml.decompositions.enable_graph()`, the following new features are availab
-  * Significantly richer variety of classical processing on mid-circuit measurement values.
-  * Using mid-circuit measurement values as gate parameters.
-
-  Functions such as the following can now be captured:
+  With `mcm_method = "deferred"`, workflows with mid-circuit measurements can be executed when program
+  capture enabled. Additionally, program capture unlocks the ability to classically process MCM values
+  and use MCM values as gata parameters.
 
   ```python
+  import jax 
   import jax.numpy as jnp
+  jax.config.update("jax_enable_x64", True)
 
   qml.capture.enable()
 
+  @qml.qnode(qml.device("default.qubit", wires=3), mcm_method="deferred")
   def f(x):
       m0 = qml.measure(0)
       m1 = qml.measure(0)
+
+      # classical processing
       a = jnp.sin(0.5 * jnp.pi * m0)
       phi = a - (m1 + 1) ** 4
 
-      qml.s_prod(x, qml.RZ(phi, 0))
+      qml.s_prod(x.astype(float), qml.RX(phi, 0))
 
       return qml.expval(qml.Z(0))
   ```
 
-* Added class `qml.capture.transforms.UnitaryToRotInterpreter` that decomposes `qml.QubitUnitary` operators
-  following the same API as `qml.transforms.unitary_to_rot` when experimental program capture is enabled.
-  [(#6916)](https://github.com/PennyLaneAI/pennylane/pull/6916)
-  [(#6977)](https://github.com/PennyLaneAI/pennylane/pull/6977)
+  ```pycon
+  >>> f(0.1)
+  Array(0.00540302, dtype=float64)
+  ```
 
-* Added a class `qml.capture.transforms.MergeAmplitudeEmbedding` that merges `qml.AmplitudeEmbedding` operators
-  following the same API as `qml.transforms.merge_amplitude_embedding` when experimental program capture is enabled.
-  [(#6925)](https://github.com/PennyLaneAI/pennylane/pull/6925)
+  Note that, with capture enabled, the number of wires given to the device must coincide with how many
+  MCMs are present in the circuit, since deferred-measurements adds one auxiliary qubit per MCM.
 
-* With program capture enabled, `QNode`'s can now be differentiated with `diff_method="finite-diff"`.
+* Quantum circuits can now be differentiated on `default.qubit` and `lightning.qubit` with 
+  `diff_method="finite-diff"`, `"adjoint"`, and `"backprop"` when program capture is enabled.
   [(#6853)](https://github.com/PennyLaneAI/pennylane/pull/6853)
-
-* Device-provided derivatives are integrated into the program capture pipeline.
-  `diff_method="adjoint"` can now be used with `default.qubit` when capture is enabled.
   [(#6875)](https://github.com/PennyLaneAI/pennylane/pull/6875)
   [(#7019)](https://github.com/PennyLaneAI/pennylane/pull/7019)
 
+  ```python
+  qml.capture.enable()
+
+  @qml.qnode(qml.device("default.qubit", wires=3), diff_method="adjoint")
+  def f(phi):
+      qml.RX(phi, 0)
+      return qml.expval(qml.Z(0))
+  ```
+
+  ```pycon
+  >>> qml.grad(f)(jnp.array(0.1))
+  Array(-0.09983342, dtype=float64, weak_type=True)
+  ```
+
+* QNode arguments can now be assigned as static. In turn, PennyLane can now smartly determine when plxpr 
```

Comment:
Might be worth linking to more information on what a "static" argument is? Maybe somewhere in JAX docs